.github/workflows/tests.yaml

```diff
@@ -0,0 +1,93 @@
+name: tests
```

Comment:
Removed image builds from the unit tests since unit tests don't run container images.